src/Database.php

```diff
@@ -314,17 +310,13 @@ public function createCollection(string $collectionName, array $options = []): a
      * getPrevious() and getEncryptedFields() methods, respectively.
      *
      * @see CreateCollection::__construct() for supported options
-     * @return array A tuple containing the command result document from creating the collection and the modified "encryptedFields" option
+     * @return array The modified "encryptedFields" option
```

Comment:
Instead of a tuple, we return the 2nd argument (the `encryptedFields`), but the return type is not modified.